### PR TITLE
Do not keep simple blocks in one line

### PR DESCRIPTION
**Pull request by ssheikin**
Before:
```if (condition) {return;}``` 

After:
```
if (condition) {
    return;
}
```

### CODE REVIEW

**Review by kokosing — COMMENTED** [review]
Can you please try to update Presto to see what it improves? You don't have to update it all, just to show the benefits.

**Comment by kokosing** [comment]
How does it work for empty methods?

**Comment by kokosing** [comment]
From:
```
    private static Expression toDoubleLiteral(double value)
    {
        if (!isFinite(value)) { return NULL_DOUBLE; }
        return new DoubleLiteral(Double.toString(value));
    }

    void x() {}

    void y() { while (true) { }if (x) return; }
```
To:
```
    private static Expression toDoubleLiteral(double value)
    {
        if (!isFinite(value)) {
            return NULL_DOUBLE;
        }
        return new DoubleLiteral(Double.toString(value));
    }

    void x() {}

    void y()
    {
        while (true) {
        }
        if (x) {
            return;
        }
    }
```

Before it was changing to:
```
    private static Expression toDoubleLiteral(double value)
    {
        if (!isFinite(value)) { return NULL_DOUBLE; }
        return new DoubleLiteral(Double.toString(value));
    }

    void x() {}

    void y()
    {
        while (true) { }
        if (x) { return; }
    }
```

**Review by kokosing — APPROVED** [review]

**Comment by findepi** [comment]
How does it work for a single line lambda, like

```
{ throw ....; }
```
?

File: IntelliJIdea2019/Airlift.xml [hunk 1]
```diff
@@ -144,7 +144,6 @@
     <option name="THROWS_LIST_WRAP" value="1" />
     <option name="EXTENDS_KEYWORD_WRAP" value="2" />
     <option name="THROWS_KEYWORD_WRAP" value="2" />
-    <option name="KEEP_SIMPLE_BLOCKS_IN_ONE_LINE" value="true" />
```
Comment (dain) [inline IntelliJIdea2019/Airlift.xml:147]: Should this be `false` instead of removed?

**Review by dain — COMMENTED** [review]
I have this disabled, so seems right to me

**Comment by kokosing** [comment]
```
        x(Object::new);
        x(() -> { return null; });
        x(() -> { throw null; });

        return new DoubleLiteral(Double.toString(value));
    }

    static void x(Supplier<?> supplier) {}
```
@findepi  The above code after and before is not changing.

File: IntelliJIdea2019/Airlift.xml (hunk 1 above)
Comment (ssheikin) [reply IntelliJIdea2019/Airlift.xml:147]: You are right, it would state explicitly our preferences.

**Comment by ssheikin** [comment]
Just for information: there are different toggles present under changed one:
```
    <option name="KEEP_SIMPLE_METHODS_IN_ONE_LINE" value="true" />
    <option name="KEEP_SIMPLE_LAMBDAS_IN_ONE_LINE" value="true" />
    <option name="KEEP_SIMPLE_CLASSES_IN_ONE_LINE" value="true" />
```

**Pull request merged**